### PR TITLE
OBPIH-4444 allow deactivation of person records

### PR DESCRIPTION
**Pull request by DaRoTP**

### CODE REVIEW

File: grails-app/controllers/org/pih/warehouse/user/UserController.groovy [hunk 1]
```diff
@@ -118,6 +118,10 @@ class UserController {
         log.info "attempt to save the user; show form with validation errors on failure"
         def userInstance = new User(params)
 
+        if (!params.hasProperty("active")) {
```
Comment (awalkowiak) [inline grails-app/controllers/org/pih/warehouse/user/UserController.groovy:121]: Is this part required if there is a default value on it?

File: grails-app/services/org/pih/warehouse/core/UserService.groovy [hunk 2]
```diff
@@ -269,6 +269,9 @@ class UserService {
                         ilike("email", "%" + term + "%")
                     }
                 }
+                if (params.status) {
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/core/UserService.groovy:272]: Why it's called `status` instead of `active`?

Comment (DaRoTP) [reply grails-app/services/org/pih/warehouse/core/UserService.groovy:272]: this same logic of calling this parameter _status_ was already used in a method `def findUsers(...)`   [on this line ](https://github.com/openboxes/openboxes/blob/f98db19ecfa175b8830f724d635d6154e4d3c9d1/grails-app/services/org/pih/warehouse/core/UserService.groovy#L293) so I just followed the convention that was already used in this situtation.

File: grails-app/controllers/org/pih/warehouse/user/UserController.groovy (hunk 1 above)
Comment (DaRoTP) [reply grails-app/controllers/org/pih/warehouse/user/UserController.groovy:121]: the default value for _active_ on **Person**  is set to `True` which is inherited by **User** but when creating a new user default value should be `False`

Comment (awalkowiak) [reply grails-app/controllers/org/pih/warehouse/user/UserController.groovy:121]: Could you add this as a comment above?

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**